sirepo/template/radia.py

```diff
@@ -418,15 +449,45 @@ def _read_or_generate(geom_id, data):
         return get_application_data(data)
 
 
-def _save_fm_sdds(name, f_data, file_path):
+def _read_solution(sim_id):
+    try:
+        return _read_path(sim_id, 'solution')
+    except KeyError:
+        # not solved yet
+        return []
+
+
+def _rot_flat_list(l, r):
+    return r.apply(numpy.reshape(l, (-1, 3)))
+
+
+def _save_field_csv(f_type, vectors, rot, file_path):
+    #pkdp('SAVE TYPE {} V {} PATH {}', f_type, vectors, file_path)
+    # rotate so the beam axis is aligned with z
+    data = ['x,y,z,' + f_type + 'x,' + f_type + 'y,' + f_type + 'z']
+    # mm -> m
+    pts = 0.001 * _rot_flat_list(vectors.vertices, rot).flatten()
```

Comment:
f_type is field type? file_path could be shortened to path. `rot` is not clear to me.  If _rot_flat_list is supposed to flatten why is `flatten` called?

Reply:
```_rotate_flat_vector_list()``` (proposed new name) does not flatten, it accepts a flat list.  We need the resultant list un-flattened to craft the sdds columns, but for csv it needs to be flat again.

I'll change ```f_type``` to ```field_type``` where it is a passed parameter